src/ErrorLog.php

```diff
@@ -0,0 +1,16 @@
+<?php
+namespace Pokedex;
+
+class ErrorLog
+{
+    private const ERROR_LOG = 'errorLog.txt';
+
+    /** This logs messages to the error log
+     * @param string $error Message can be user defined or accessed from Exception object
+     */
+    public static function log(string $errorMessage) {
+        $errorLog = fopen(self::ERROR_LOG, 'a');
```

Comment:
Does not work if errorLog.txt does not already exist.

Reply:
changed to mode a+ in 66bc114